spec/concierge_spec.rb

```diff
@@ -0,0 +1,124 @@
+require_relative 'spec_helper'
+require 'pry'
+
+describe "Concierge class" do
+
+  describe "Initialize" do
+    before do
+      @concierge = Hotel::Concierge.new
+    end
+
+    it "is an instance of Concierge" do
+      expect(@concierge).must_be_kind_of Hotel::Concierge
+    end
+   
+    it "loads a list of all rooms" do
+      expect(@concierge.all_rooms).must_be_kind_of Array
+    end
+  end
+  
+  
+  describe "See all rooms method" do
+    before do
+      @concierge = Hotel::Concierge.new
+    end    
```

Comment:
Since this `before` block is repeated, it might make sense to move it out to the top-most `describe`.